openshift/templates/aries-mediator-agent/container/Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM ghcr.io/hyperledger/aries-cloudagent-python:py3.9-0.10.4
+FROM ghcr.io/openwallet-foundation/acapy-agent:py3.12-1.1.0
 
 # ========================================================================================================================================
 # Install Plugins
 # ----------------------------------------------------------------------------------------------------------------------------------------
 # Redis PQ
-RUN pip install --no-cache-dir git+https://github.com/hyperledger/aries-acapy-plugins.git@main#subdirectory=redis_events
+RUN pip install --no-cache-dir git+https://github.com/openwallet-foundation/acapy-plugins.git@main#subdirectory=redis_events
 # Firebase Push Notifications
-RUN pip install --no-cache-dir -e "git+https://github.com/hyperledger/aries-mediator-service.git@main#egg=firebase_push_notifications&subdirectory=acapy/plugins/firebase_push_notifications"
-# ========================================================================================================================================
\ No newline at end of file
+RUN pip install --no-cache-dir git+https://github.com/openwallet-foundation/acapy-plugins.git@main#subdirectory=firebase_push_notifications
```

Comment:
We should probably install the tagged version of the plugin rather than the one from main. You can see an example of how we do it [here](https://github.com/bcgov/traction/blob/693d1b458b6d437801833b028b72f1c82d7e0250/plugins/docker/Dockerfile#L33-L36)